package.json

```diff
@@ -145,6 +145,7 @@
     "file-saver": "^1.3.3",
     "immutable": "^3.8.2",
     "jquery": "^3.2.1",
+    "json-formatter-js": "^2.2.1",
```

Comment:
this could be removed right, as we don't use it?

Reply:
Correct